src/components/common/IntegerInput/IntegerInput.scss

```diff
@@ -0,0 +1,5 @@
+input::-webkit-outer-spin-button,
+input::-webkit-inner-spin-button {
+  -webkit-appearance: none;
+  margin: 0;
+}
```

Comment:
Whats happening here ?

Reply:
Removed, that was for when field was a number field.